browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2366,17 +2366,6 @@ export class CommentSection extends app.definitions.canvasSectionObject {
 		this.disableLayoutAnimation = true;
 		var comment;
 		if (Comment.isAnyEdit()) {
-			this.map.uiManager.showConfirmModal(
```

Comment:
So probably we should detect when we initiate import or not (as optional param?)
Then we can show the dialog only when needed

Reply:
We have below lines:
```
			if (refreshAnnotation)
				app.socket.sendMessage('commandvalues command=.uno:ViewAnnotations');
```

I guess that re-import breaks things. My take is to keep this modal out for now. We have onACKComment and it also seems to do the job.

Reply:
If it is only for now, then would be good to keep strings so we will not lost translations already done.
Could you move it to separate method (not used) - if that's the case. If not we can delete.

Reply:
In fact, i think refreshAnnotation boolean is somehow needed and it re-imports all the comments. In that case, "importComments" is not something that is called at initialization. All the comments are re-imported when that function is called. Including the one we just created and still editing. So i think we can't have that confirmation modal there.

We need to remove it. Then we can have a better look at the function calls and add something else i guess.